using_images/other_images/jenkins.adoc

```diff
@@ -429,7 +429,7 @@ container hosts using Kubernetes and {product-title}.
 
 To use the Kubernetes plug-in, {product-title} provides five images suitable
 for use as Jenkins agents: the *_Base_*, *_Maven_*, and *_Node.js_* images. See
-xref:jenkins_slaves.adoc#overview[Jenkins Agents] for more information.
+xref:jenkins_agents.adoc#overview[Jenkins Agents] for more information.
 
 NOTE:  the jenkins-slave-maven-* and jenkins-slave-nodejs-* images are being
```

Comment:
Not part of this PR, but I wonder if you could change 'the' in this note to 'The', please?